lib/boku/forms.py

```diff
@@ -55,6 +54,19 @@ def clean(self):
                                                 cleaned_data.get('currency')))
         return cleaned_data
 
+    def clean_sig(self):
+        # Check that the signature was from Boku. This will raise an error
+        # if the signing is incorrect, but not check the transaction_id is
+        # valid.
```

Comment:
This reads awkwardly to me. Maybe: "This will raise an error if the signature is incorrect but it does not validate the transaction_id."